### PR TITLE
Exposure detection information i18n

### PR DESCRIPTION
**Pull request by ChristianKienle**
# What to test
- Ensure that the new information dialog is shown after onboarding - only once.

- Make sure that you see the new text for active tracing on the home screen (in low risk mode).

- Make sure that you see the same info on the detail screen + the new dynamic information:

| Top  | Further Down  |
|---|---|
|   |   |

- Check the exposure setting screen:

### CODE REVIEW

File: src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENATests.xcscheme [hunk 1]
```diff
@@ -57,6 +57,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
+      region = "DE"
```
Comment (haosap) [inline src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENATests.xcscheme:61]: the set is on purpose?

**Review by haosap — APPROVED** [review]
Thanks.

File: src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENATests.xcscheme (hunk 1 above)
Comment (steins-code) [reply src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENATests.xcscheme:61]: guess you reset this change ;)

File: src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.strings [hunk 2]
```diff
@@ -97,13 +97,51 @@
...
+"ExposureDetection_ActiveTracingSection_Subtitle" = "Bu dönem hesaplamaya dahil edildi";
+
+/*
+ There are two paragraphs in this section. The first one just contains text…
+ */
+"ExposureDetection_ActiveTracingSection_Text_Paragraph0" = "Enfeksiyon riskiniz yalnızca maruz kalma günlüğünün etkin olduğu dönemler için hesaplanabilir. Bu nedenle günlüğe kaydetme özelliğinin sürekli etkin kalması gerekir.";
+
+/*
+ … the second paragraph contains two columns. On the left there is our active
+ tracing indicator. On the right column there is just text that explains what
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.strings:111]: do we need these comments?;)

**Review by steins-code — APPROVED** [review]

**Pull request merged**